acme/setup.py

```diff
@@ -53,7 +53,7 @@
     version=version,
     description='ACME protocol implementation in Python',
     url='https://github.com/letsencrypt/letsencrypt',
-    author="Let's Encrypt Project",
+    author="Electronic Frontier Foundation",
```

Comment:
Well, there were other contributors as well. Shouldn't everybody be captured here (and in other setup.py files)?